software/fosscord-server/egg-fosscord.json

```diff
@@ -0,0 +1,41 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-11-26T10:45:23-05:00",
+    "name": "Fosscord",
+    "author": "chris@chrischromeyt.com",
+    "description": "https:\/\/github.com\/fosscord\/fosscord-server",
+    "features": null,
+    "images": [
+        "ghcr.io\/parkervcp\/yolks:nodejs_17"
```

Comment:
Would rather stick to LTS releases, meaning 14 or 16. The minimum requirement for fosscord is node 14.